packages/synapse-react-client/src/components/RowDataTable/RowDataTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { SelectColumn } from '@sage-bionetworks/synapse-types'
+import { LabelLinkConfig } from '../CardContainerLogic'
+import { StyledTableContainer } from '../styled/StyledTableContainer'
+import { SynapseCardLabel } from '../GenericCard'
+
+export type RowDataTableProps = {
+  /** Array of string values corresponding to the data for a row */
+  rowData: string[]
+  /** The headers for the columns in the table */
+  headers: SelectColumn[]
+  /** List of column names to display as labels */
+  labels: string[]
```

Comment:
How about renaming this and slightly modifying logic to something like...
```suggestion
  /** Optional list of column names whose data should be displayed in the table. If not provided, all columns with data will be shown. */
  displayedColumns?: string[]
```